spec/acceptance/firewall_attributes_exceptions_spec.rb

```diff
@@ -19,9 +19,8 @@
 
   # --bytecode is only supported by operatingsystems using nftables (in general Linux kernel 3.13, RedHat 7 (and derivates) with 3.10)
   # Skipping those from which we know they would fail.
-  describe 'bytecode property', unless: (os[:family] == 'redhat' && os[:release][0] <= '6') ||
-                                        (os[:family] == 'sles' && os[:release][0..1] <= '11') ||
-                                        (fetch_os_name == 'oraclelinux' && os[:release][0] <= '7') ||
```

Comment:
may be we keep oraclelinux condition as it is, as we have only 1 version i.e. 7, once we add new version that time we can evaluate it?

Reply:
Yes, we'll keep it as is.